torch/csrc/jit/codegen/cuda/fusion.cpp

```diff
@@ -428,6 +496,34 @@ StmtNameType Fusion::registerStatement(Statement* stmt) {
   return UNINITIALIZED_STMTNAMETYPE;
 }
 
+StmtNameType Fusion::registerLoweredVal(Val* val) {
+  TORCH_CHECK(val->fusion() == this);
```

Comment:
nit: Should this be `TORCH_INTERNAL_CHECK`?

Reply:
Sure, I'll change it.